Quiz/src/jsx/components/Answer.jsx

```diff
@@ -1,32 +1,30 @@
 import React from "react";
 
-class Answer extends React.Component{
-    constructor(props) {
-        super(props);
+const Answer = props => (
+    <form onSubmit={props.onClickCallback}>
+        {props.options.map((option,i) => (
+            <div className={`option${i}`} key={i}>
+                <input
+                    type="radio"
+                    name='rad'
+                    id={`option_${i}`}
+                    onChange={() => {
+                        props.handleRadio(i)
```

Comment:
I prefer using anonymous event handlers in the view and calling function references within that – this avoids the necessity of re-binding class methods elsewhere because we explicitly avoid the `this` re-binding that comes from event handler assignment. Previously, the parent component's `this` binding for `handleRadio` was over-written by the binding here.